utils/EA/fitness.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.model_selection import cross_val_score
+
+from .population import phenotype
+
+def fitness(sick_data, sick_labels, healthy_data, healthy_labels):
+    def fitness_(indiv):
+        pheno = phenotype(indiv)
+        # select features and only pass this data to evaluate
+
+        return evaluate(sick_data[:, pheno], sick_labels, healthy_data[:, pheno], healthy_labels)
+    return fitness_
+
+def evaluate(sick_data, sick_labels, healthy_data, healthy_labels):
+    clf = DecisionTreeClassifier()
+
+    if sick_data.shape[1] > 10:
+        return -10
```

Comment:
Also Lukas proposed to return Double.minValue, and I second that

Reply:
Yep. The fitness function needs a lot of more experimentation. We should talk about this together and than think about the possible range of values and than return the minimum of that here/ or Double.minValue.
Just want to add that it is sometimes useful for EAs to also take "bad" individuals (e.g. when it is stuck in local optima). There it could make sense to increase the diversity and start-over